packages/legacy/src/common/constants.ts

```diff
@@ -17,7 +17,7 @@ export const ENV: IEnvVars = {
 };
 
 if (ENV.NODE_ENV === 'development') {
-  console.info('forklift-console-plugin ENV:', JSON.stringify(ENV));
+  console.info('console-plugin ENV:', JSON.stringify(ENV));
```

Comment:
changed to `console-plugin` so users will not think it's the plugin name and will look at `ENV.PLUGIN_NAME` value instead.